src/otx/algo/instance_segmentation/rtmdet_inst.py

```diff
@@ -5,44 +5,62 @@
 
 from __future__ import annotations
 
-from copy import deepcopy
-from typing import TYPE_CHECKING, Literal
+from typing import TYPE_CHECKING
 
-from otx.algo.utils.mmconfig import read_mmconfig
+from omegaconf import DictConfig
+
+from otx.algo.detection.backbones.cspnext import CSPNeXt
+from otx.algo.detection.heads.base_sampler import PseudoSampler
+from otx.algo.detection.heads.distance_point_bbox_coder import DistancePointBBoxCoder
+from otx.algo.detection.heads.dynamic_soft_label_assigner import DynamicSoftLabelAssigner
+from otx.algo.detection.heads.point_generator import MlvlPointGenerator
+from otx.algo.detection.losses.cross_entropy_loss import CrossEntropyLoss
+from otx.algo.detection.losses.dice_loss import DiceLoss
+from otx.algo.detection.losses.gfocal_loss import QualityFocalLoss
+from otx.algo.detection.losses.iou_loss import GIoULoss
+from otx.algo.detection.necks.cspnext_pafpn import CSPNeXtPAFPN
+from otx.algo.instance_segmentation.mmdet.models.dense_heads.rtmdet_ins_head import RTMDetInsSepBNHead
+from otx.algo.instance_segmentation.mmdet.models.detectors.rtmdet import RTMDet
 from otx.core.config.data import TileConfig
 from otx.core.exporter.base import OTXModelExporter
-from otx.core.exporter.mmdeploy import MMdeployExporter
+from otx.core.exporter.native import OTXNativeModelExporter
 from otx.core.metrics.mean_ap import MaskRLEMeanAPCallable
 from otx.core.model.base import DefaultOptimizerCallable, DefaultSchedulerCallable
 from otx.core.model.instance_segmentation import MMDetInstanceSegCompatibleModel
+from otx.core.model.utils.mmdet import DetDataPreprocessor
 from otx.core.schedulers import LRSchedulerListCallable
 from otx.core.types.label import LabelInfoTypes
-from otx.core.utils.utils import get_mean_std_from_data_processing
 
 if TYPE_CHECKING:
+    import torch
     from lightning.pytorch.cli import LRSchedulerCallable, OptimizerCallable
+    from torch.nn.modules import Module
 
     from otx.core.metrics import MetricCallable
 
 
-class RTMDetInst(MMDetInstanceSegCompatibleModel):
-    """RTMDetInst Model."""
+class MMDetRTMDetInstTiny(MMDetInstanceSegCompatibleModel):
```

Comment:
I'm not sure `MMDetRTMDetInstTiny` is proper because `mmdet` is here.
What do you think?

Reply:
If my understanding is correct, this will eventually lead to the deprecation of `MMDetInstanceSegCompatibleModel`, so this name change should be fine. @eugene123tw Please confirm.

Reply:
@harimkang @sungchul2  yes, that's right. I'm decoupling `MMDetInstanceSegCompatibleModel` now using some of Sungchul's draft PR and it will not use any mmdet data-related implementation (PolygonMask, BitMask, DetDataSample, etc).

Reply:
If decoupling is finished, all ins seg model otx provide should inherit ExplainableOTXInstanceSegModel